python/ray/reporter.py

```diff
@@ -222,10 +227,10 @@ def perform_iteration(self):
     def run(self):
         """Publish the port."""
         thread_pool = futures.ThreadPoolExecutor(max_workers=10)
-        server = grpc.server(thread_pool, options=(("grpc.so_reuseport", 0), ))
```

Comment:
Should we still be using `grpc.so_reuseport`?

Reply:
I am not sure why we need this option actually. Do we need it to run thread pool? It seems like the option is to bind multiple processes to a single port, but it looks like we only have a single reporter.py process? I might be missing something.